ArduCopter/GCS_Mavlink.cpp

```diff
@@ -1125,7 +1125,7 @@ void GCS_MAVLINK_Copter::handleMessage(const mavlink_message_t &msg)
         bool yaw_rate_ignore = packet.type_mask & MAVLINK_SET_POS_TYPE_MASK_YAW_RATE_IGNORE;
 
         // exit immediately if acceleration provided
```

Comment:
we should update this comment to, "// exit immediately if neither position nor velocity is provided"